common/static/common/templates/discussion/forum-action-vote.underscore

```diff
@@ -2,7 +2,7 @@
     <span aria-hidden="true" class="display-vote is-hidden" >
       <span class="vote-count"></span>
     </span>
-    <a href="#" class="action-button action-vote" role="checkbox" aria-checked="false">
+    <button class="btn-link action-button action-vote" role="checkbox" aria-checked="false">
```

Comment:
Good question. It is kind-of a toggle, but it seems more like an action to me. Can I just remove the role and aria-checked attributes?